source/content/guides/composer-convert.md

```diff
@@ -89,6 +88,9 @@ This document is for you if you meet the following criterion:
 
   Anything other than "no updates available" and you will need to apply the updates either by command line or via the Pantheon dashboard before continuing.
 
+- You are not using a package/library manager (like "Ludwig") that is not COMPOSER.
+
+  Some installs have used a package manager (e.g. [Ludwig](https://www.drupal.org/project/ludwig)). If you meet all the other requirements, you can cd to your root folder and run `composer update` to get "Back on the composer ranch" and add your modules and themes to the composer file as below.
 
```

Comment:
@greg-1-anderson plz review

Reply:
I would not recommend that we advise they do this before they start; if they are using Ludwig, then just running `composer update` could put their site in a bad state. Instead of this, we should tell them to disable Ludwig after they create the multidev, but before they change any of the code in the multidev.

In order to do this, though, we'll need to change the instructions such that they create the multidev sooner, because right now we don't create the multidev until after we make the new branch, and at that point it's too late to disable ludwig.

Reply:
Edward and I have agreed that Ludwig is, for the moment, out of scope. We're simply going to say "If you're using it disable it." and make changes to those instructions going forward. I think that's what we agreed, please correct me if I'm in err, @EdwardAngert

Reply:
n.b. tho if you say "If you're using it disable it," you have to do that in the multidev, because if you disable Ludwig on a site that's using it, you break the site, so that has to happen in the work environment, not in dev.

Reply:
for this version of the doc, we've settled on suggesting that users not use the doc if they're using Ludwig. I'll mark this resolved and we can revisit how to include Ludwig-specific instructions in a later docs release